docs/Reference/Hooks.md

```diff
@@ -827,18 +827,18 @@ consider creating a custom [Plugin](./Plugins.md) instead.
 ## Diagnostics Channel Hooks
 
 > **Note:** The `diagnostics_channel` is currently experimental on Node.js, so
-> its API is subject to change even in semver-patch releases of Node.js. For
-> versions of Node.js supported by Fastify where `diagnostics_channel` is
-> unavailable, the hook will use the
-> [polyfill](https://www.npmjs.com/package/diagnostics_channel) if it is
-> available. Otherwise, this feature will not be present.
-
-Currently, one
-[`diagnostics_channel`](https://nodejs.org/api/diagnostics_channel.html) publish
-event, `'fastify.initialization'`, happens at initialization time. The Fastify
-instance is passed into the hook as a property of the object passed in. At this
-point, the instance can be interacted with to add hooks, plugins, routes, or any
-other sort of modification.
+> its API is subject to change even in semver-patch releases of Node.js. As some
+> versions of Node.js are supported by Fastify where `diagnostics_channel` is
+> unavailable, or with an incomplete feature set, the hook uses the
+> [dc-polyfill](https://www.npmjs.com/package/dc-polyfill) package to provide a
+> polyfill. Feel free to either use this package or `diagnostics_channel` itself
+> in your application.
```

Comment:
What benefit is the final sentence providing to the Fastify docs?

```suggestion
> polyfill.
```

Reply:
An application would benefit if it needs to run on a wide range of Node.js versions. But, since app owners have more control over Node.js, the point is moot. I'll apply your suggested removal.